pyproject.toml

```diff
@@ -9,6 +9,7 @@ authors = [
     { name = "Florian Dobener", email = "florian.dobener@physik.hu-berlin.de" },
```

Comment:
This email does not work anymore, no? Maybe use his github email?

Reply:
I think we can just remove the email.